### DISCUSSION TITLE
Is there a screen or function to manage user accounts?

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#256] [Feature Requests & Suggestions]
I think it would be nice to be able to create a user with admin privileges

**Comment by danny-avila** [comment]
This is not a priority but may come in the future. Look into mongo-express for some of this functionality right away

**Reply by tttll1091** [reply]
I hope

**Reply by PedroGruvhagen** [reply]
hello!
is there a way to make an user admin @danny-avila ?
